internal/bgp/frr/frr.go

```diff
@@ -219,12 +219,21 @@ func (sm *sessionManager) createConfig() (*frrConfig, error) {
 		config.Loglevel = frrLogLevel
 	}
 
-	for _, s := range sm.sessions {
+	// to get the sessions in some predictable order, we need to sort the keys
+	// this is needed for testing purpose
+	var sessionKeys []string
```

Comment:
you can use sortmap here, or have a variation for pointers
 https://github.com/metallb/metallb/blob/651427b19fcc9771f03bf5bff5eb3305229d0c10/internal/bgp/frr/frr.go#L450